gcloud/language/document.py

```diff
@@ -0,0 +1,97 @@
+# Copyright 2016 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Definition for Google Cloud Natural Language API documents.
+
+A document is used to hold text to be analyzed and annotated.
+"""
+
+
+DEFAULT_LANGUAGE = 'en'
+"""Default document language, English."""
+
+
+class Encoding(object):
+    """Document text encoding types."""
+
+    NONE = 'NONE'
+    """Unspecified encoding type."""
+
+    UTF8 = 'UTF8'
+    """UTF-8 encoding type."""
+
+    UTF16 = 'UTF16'
+    """UTF-16 encoding type."""
+
+    UTF32 = 'UTF32'
+    """UTF-32 encoding type."""
+
+
+class Document(object):
+    """Document to send to Google Cloud Natural Language API.
+
+    Represents either plain text or HTML, and the content is either
+    stored on the document or referred to in a Google Cloud Storage
+    object.
+
+    :type content: str
+    :param content: (Optional) The document text content (either plain
+                    text or HTML).
+
+    :type gcs_url: str
+    :param gcs_url: (Optional) The URL of the Google Cloud Storage object
+                    holding the content.
```

Comment:
Should the docs for `content` and `gcs_uri` indicate that they are in a `OneOf` relationship?